perl-alien/alienfile

```diff
@@ -1,9 +1,11 @@
 use alienfile;
 
+my $libyamlscript_version = '0.1.34';
+
 # Always use a share install
 # We cannot use a system install, because we need to know exactly
 # which version we are installing.
-probe sub { 'share' };
+use constant probe => 'share';
```

Comment:
These are not equivalent.

It used to be
```perl
probe sub { 'share' };
```

But now it is
```
sub probe () { 'share' }
```

Reply:
oops, my bad :)